libs/types/src/interpretation.ts

```diff
@@ -45,11 +49,24 @@ export const InterpretedBmdPageSchema: z.ZodSchema<InterpretedBmdPage> =
     votes: VotesDictSchema,
   });
 
+export interface UnmarkedWriteIn {
+  contestId: ContestId;
+  optionId: WriteInId;
+  score: number;
+}
+
+export const UnmarkedWriteInSchema: z.ZodSchema<UnmarkedWriteIn> = z.object({
+  contestId: ContestIdSchema,
+  optionId: WriteInIdSchema,
+  score: z.number(),
+});
+
 export interface InterpretedHmpbPage {
   type: 'InterpretedHmpbPage';
   ballotId?: BallotId;
   metadata: HmpbBallotPageMetadata;
   markInfo: MarkInfo;
+  unmarkedWriteIns?: UnmarkedWriteIn[];
```

Comment:
I'm not quite sure what the best way to represent them here is.

The way we represent marks is currently confusing - we keep both the raw mark data and the interpreted mark data (votes) in the return data of the interpreter.

We could do the same for `unmarkedWriteIns` - an `unmarkedWriteInInfo` and a `unmarkedPossibleWriteInVotes`. I'm not sure which direction we're heading at this layer - exposing more detail or exposing less. I'm also not sure if changes to the interpreter re: parameter-izable write-in areas will require more information to be passed here, or if all that necessary information will be caught in the layout.

Anyway, welcoming feedback on this data representation.